### DISCUSSION TITLE
[Help]: How can i make the carousel undraggable?

### DISCUSSION THREAD

**Discussion by Zeken0** [davidjerleke/embla-carousel#977] [Help] [answered]
### Summary

Like the title says I'm trying to make the carousel undraggable while the auto-scroll plugin is on.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@Zeken0 this seems to be a duplicate of:

- https://github.com/davidjerleke/embla-carousel/discussions/918#discussioncomment-9804110

**Reply by Zeken0** [reply]
worked, thanks man.
